reducers/plugins/index.js

```diff
@@ -153,6 +153,28 @@ function components(state = {}, action) {
     }
 }
 
+function hooks(state = {}, action) {
```

Comment:
We actually don't need a new reducer for this, we've been using the components reducer for practically everything. e.g. see the [FilesWillUploadHook](https://github.com/mattermost/mattermost-webapp/blob/master/plugins/registry.js#L220)